containers/home.tsx

```diff
@@ -1,66 +1,89 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import VideoCardList from '../components/videoCardList/videoCardList';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+
+interface iVideo {
+  cards?: [{ title: string; alt: any; src: any }];
```

Comment:
Here with the `iCard` interface you can do something like:

```jsx
interface iCard {
  title: string,
  alt?: string,
  src?: string
}
```

Where `alt` and `src` are optional values, if those values are not optional then just remove the question mark `?`

Reply:
`cards?: Array<iCard>`